src/jepsen/redis/core.clj

```diff
@@ -161,8 +164,11 @@
    [nil "--tcpdump" "Create tcpdump logs for debugging client-server interaction"
     :default false]
 
-   ["-v" "--version VERSION" "What version of Redis should we test?"
-    :default "6.0.3"]
+   [nil "--redis-version VERSION" "What version of Redis should we test?"
+    :default "unstable"]
```

Comment:
Ditto here--up to y'all, but this might make it difficult for you to do regression testing

Reply:
Yes, I considered that but having a fixed stale version is also not the best. I'll just drop the default to make everything explicit.

Reply:
That's fine! Or you can do unstable here! Either way, just wanted to check. :-)